packages/tools/examples/labelmapSegmentLocking/index.ts

```diff
@@ -259,16 +255,28 @@ async function run() {
     [viewportId1, viewportId2, viewportId3]
   );
 
-  // // Add the segmentation representation to the toolgroup
-  await segmentation.addSegmentationRepresentations(toolGroupId, [
+  // // Add the segmentation representation to the viewport
+  await segmentation.addSegmentationRepresentations(viewportId1, [
```

Comment:
This feels a bit clunky, like one should be able to add the representations automatically based on the volume/stack being shown in a viewport, and have them USUALLY be the same.  What about:
await segmentation.addSegmentationRepresentations( viewportIds, [ ....] );
so that the common case of mapping them for all viewports is easily handled.  I agree, it is important to be able to add them differently, but adding them in bulk also seems important.

Reply:
I guess it feels like one should be able to assign a set of key/value pairs identifying which reps get added where, and then they just get auto-added when configured.  Something like a viewport property that defines part of it, and the volume or group (stack) id as the other half.